tests/tests_pytorch/strategies/test_sharded_strategy.py

```diff
@@ -125,27 +125,7 @@ def test_ddp_sharded_strategy_fit_ckpt_path(tmpdir):
     trainer.fit(model, ckpt_path=checkpoint_path)
 
 
-@pytest.mark.skip(reason="Not a critical test, skip till drone CI performance improves.")  # todo
-@pytest.mark.skip(reason="Currently unsupported restarting training on different number of devices.")
-@RunIf(min_cuda_gpus=2, skip_windows=True, fairscale=True)
-def test_ddp_sharded_strategy_fit_ckpt_path_downsize_gpus(tmpdir):
```

Comment:
> Currently unsupported restarting training on different number of devices.

have you tested it?

Reply:
if there's some issue, we should rather raise errors in the framework if possible?

Reply:
I am interested in checking the error here. But let's remove this for now since it's redundant already and I will investigate it next week?

Reply:
We would want to get rid of this test anyways regardless of whether this works or not.

If it does, it should have proper assertions
If it doesn't, it should test the exception raised by the library.

Let me open an issue to track it